.github/workflows/helm-release.yaml

```diff
@@ -8,8 +8,28 @@ on:
       - '.github/workflows/helm-release.yaml'
```

Comment:
If we merge this PR to the main, this workflow will be triggered again, will it fail due to 1.4.2 is already present?

Reply:
ah yes, that's a good point. I'm not about the answer but it's not ideal is it

Reply:
We can merge and see, it doesn't fail anything except the helm-release workflow. That should be fine.